zavod/zavod/helpers/names.py

```diff
@@ -326,36 +341,15 @@ def review_names(
         context: The current context.
         entity: The entity to apply names to.
         string: The raw name(s) string.
-        alias: If this is known to be an alias and not a primary name.
-        lang: The language of the name, if known.
     """
-    strings = [s for s in strings if s]
-
-    if not strings:
+    non_blank_strings = [s for s in strings if s]
+    if not non_blank_strings:
         return None
 
-    if settings.CI or not any(is_name_irregular(entity, s) for s in strings):
-        prop = "alias" if alias else "name"
-        for string in strings:
-            entity.add(prop, string, lang=lang)
+    if settings.CI or not any(is_name_irregular(entity, s) for s in non_blank_strings):
```

Comment:
(nit) not really part of this, but might be nice to do the `settings.CI` check last just so that as much code as possible runs in CI mode?